tests/app/main/test_suppliers.py

```diff
@@ -409,11 +409,13 @@ def test_should_be_an_error_if_duns_number_in_use(self, data_api_client):
         res = self.client.post(
             "/suppliers/duns-number",
             data={
-                'duns_number': "9999999999"
+                'duns_number': "123456789"
             }
         )
         assert_equal(res.status_code, 400)
-        assert_true("DUNS Number must be 9 digits" in res.get_data(as_text=True))
+        page = res.get_data(as_text=True)
+        assert_true("A supplier account already exists with that DUNS number" in page)
```

Comment:
Ordinarily I'd agree but these kinds of tests are testing whether some text appears in the entire HTML payload. The standard `assert_in` error message spits out the whole page making it really hard to read.